sections/quick-order-list.liquid

```diff
@@ -128,7 +128,10 @@
         {% comment %} Populated by JS {% endcomment %}
       </span>
     {%- else -%}
-      <div class="quick-order-list__total gradient" id="quick-order-list-total">
+      <div
+        class="quick-order-list__total{% if product.quantity_price_breaks_configured? %} has-volume-pricing{% endif %} gradient"
```

Comment:
They could have volume pricing or qty rules. We should check for both

(qty popover will show if there are only qty rules as well)